dotnet/Service/appsettings.json

```diff
@@ -34,14 +34,18 @@
       "VectorDbTypes": [
         "AzureCognitiveSearch"
       ],
+      // Note: keep the list empty in this file, to avoid unexpected merges
```

Comment:
nit: Could we mention this in a trouble shooting guide somewhere with examples? We have arrays in other places in the settings.

Reply:
I'll try to think about a place where we could add similar notes. It's something about .NET and not specific to this project, e.g. some people already know about this - I left the note just in case one wonders why the default is empty